app/services/katello/host/profiles_uploader.rb

```diff
@@ -6,15 +6,20 @@ def initialize(profile_string:, host: nil)
         @host = host
       end
 
+      # rubocop:disable Metrics/MethodLength
       def upload
         profiles = JSON.parse(@profile_string)
         #free the huge string from the memory
         @profile_string = 'TRIMMED'.freeze
         if @host.nil?
           Rails.logger.warn("Host was not specified; continuing")
+          return
         elsif @host.content_facet.nil? || @host.content_facet.uuid.nil?
           Rails.logger.warn("Host with ID %s has no content facet; continuing" % @host.id)
-        elsif profiles.try(:has_key?, "deb_package_profile")
+          return
```

Comment:
Two things:
1. since we do not do anything with it, should the message not end with `skipping`?
2. given that we return `true` at the end, should we explicitly return `false` here? Although it is not evaluated anywhere, yet